docs/pages/docs/07-workbase/visualiser.md

```diff
@@ -0,0 +1,198 @@
+
+### Investigate a Node
+
+A single click on any node in the knowledge graph brings up a tool tip of information about the node at the top right hand side of the screen. The information displayed includes the ID of the node, its type and any attributes associated with it, as shown in the figure below.
+
+![Single click](/images/workbase/node_panel.png)
+
+A double click on a node will bring up the relationships associated with the node, as shown below.
+
+![Double click](/images/workbase/double_click.png)
+
+Holding shift and double clicking on a node also brings up the attributes associated with it, displaying them in the knowledge graph, as shown below.
+
+![shift double click](/images/workbase/shift_click.png)
+
+You may also select multiple nodes by holding Cmd/Ctrl and clicking on nodes.
+
+### Change the Display
+
+Selecting `Node Settings` in the context menu opens the node settings panel on the lower left hand side of the screen. You can use this to show different information about a node on the graph and change the colour of the nodes.
+
+![Single click](/images/workbase/node_settings_panel.png)
+
+### Query Settings
+
+Query Settings can be accessed under the cog icon at the right hand of the horizontal icon set. The following settings are available:
+
+![Person query](/images/workbase/query_settings.png)
+
+#### Limit Query 
+
+You can change the offset and limit on the number of results returned by editing the value directly in the submission form, or by adjusting the Query Limit setting.
+
+#### Limit Neighbour  
+
+Allows the user to limit the nodes loaded when double clicking or shift + double clicking on a node.
+
+#### Autoload Role Players
+
+You can enable or disable loading role players with relationships.
+
+### Explain an Inferred Concept
+
+If a nodes is inferred, you can select the `Explain` option from the context menu and get the explainaion of how it is inferred. e.g if the cousin relationship is inferred, selecting explain would give us the following:
+
+![explain](/images/workbase/explain_cousin.png)
+
+### Shortest Path
+
+Select two nodes and select the `Shortest Path` option in the context menu to display the shortest path between the two nodes if it exists.
+
+![explain](/images/workbase/shortest_path.png)
+
+### Graph Data
+
+The tool tip on the bottom right of the graph will display the number of nodes and edges in the graph.
+
+![canvas data](/images/workbase/canvas_data.png)
+
+### Commands 
+
+Selecting the information icon on the bottom left of the graph opens a modal containing all the ways to interact with the graql editor and graph.
+
+![commands icon](/images/workbase/commands_icon.png)
+
+## Manage Settings
+
+### Engine Settings
+
+Allows user to configure the host and port of Grakn.
+
+![engine settings](/images/workbase/engine_settings.png)
+
+### Preference Settings
+
+Allows user to clear all saved preferences.
```

Comment:
quick list of what are the preferences: fav queries, limit query, colours on node and node labels